sys/include/iec62056.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2023 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_iec62056 IEC62056
+ * @ingroup     sys
+ * @brief       Data exchange for meter reading, tariff and load control
+ *
+ * @{
+ *
+ * @file
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ */
+
+#ifndef IEC62056_H
+#define IEC62056_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef __cplusplus
+}
+#endif
```

Comment:
If this file is empty, why have it?

Reply:
It was useful once, but now it is just there for the doxygen grouping. I'll rework it a bit and remove the file

Reply:
replaced with a doc.txt in `sys/iec62056` for some documentation and a doxygen group.